packages/components/src/components/tooltip/_tooltip.scss

```diff
@@ -398,6 +399,28 @@
       }
     }
 
+    // Tooltips without interactive elements, need to be sequentially and click focusable so screen reader users
+    // get the tooltip's content and we focus the content container instead of the tooltip root so that
+    // focus-wrap doesn't get triggered
+    .#{$prefix}--tooltip__content[tabindex='0'] {
```

Comment:
just to confirm I understand correctly: in order to avoid triggering the focus trap on tooltips without interactive elements in the tooltip content, `tabindex="0"` needs to be set on `.bx--tooltip__content` to allow sequential keyboard focus and click focus, is that right?

do we need to add any JS to automatically add/remove `tabindex` from `.bx--tooltip__content` if the tooltip does not contain an interactive element?

Reply:
@emyarod You are correct with your understanding. In the original PR to set up tooltips with non-interactive elements (this PR is to fix a bad merge between tooltip PRs that broke this functionality) it was mentioned that it should be up to the developer to include the attribute.  Given now that the tabindex is always included (either 0 or -1), I would not be opposed to adding it to the JS to make it easier for the developers.

Reply:
I think it would be helpful to add that tabindex switching functionality in our tooltip JS but interested in hearing others' thoughts cc @asudoh

Reply:
@emyarod I did add it to the JavaScript and I added tests for it.

Reply:
Late on this thread, and need a refresher. This is for interactive tooltip, is it correct? If so, does it really make sense to mark the element sequentially-focusable in DOM given the menu is floating, where we cannot make the focus sequence in DOM match the one user sees visually, and the content node is the only focusable node in the floating menu? Note that we "simulate" sequential focus by tracking focus events.

Reply:
@asudoh It is for interactive tooltip and the "simulated" sequential focus did not work for screen reader (SR) users. You have to actually move the focus in to the dialog in order for the SR to announce the content.  When there is not interactive elements within the tooltip to focus on nothing was being read which is why we have the content actually focused.  I've made this change in a previous PR (which was approved and merged). This PR is to fix a bad merge between 2 PRs that overwrote this functionality.

Reply:
that's true, if there is an element with a higher tabindex before the floating menu in the DOM, the tab order might not match what the user expects after launching the tooltip right?

Reply:
Sorry for possibly not catching the topic before. What I understand/recall is that the floating menu code puts the focus programmatically. Did you find it differently?

Reply:
@asudoh Floating menu does support moving the focus but it needs an element to focus which is why we have the `tabindex="0"` on the element so that we know there is something to focus in the tooltip.  This PR does not change any of the JavaScript logic that moves the focus, just ensures there is something to focus otherwise "nothing" gets focus and screen reader user doesn't get the content.